### PR TITLE
#166790041 Implements setup of React environment with webpack

### PR DESCRIPTION
**Pull request by mnzube**
#### What does this PR do?
Sets up an empty react environment for implementation of Authors Haven
#### Description of Task to be completed?
- Added react, babel and webpack packages
- Added webpack configuration
#### How should this be manually tested?
clone the repository with https://github.com/andela/ah-jawans-frontend.git
run `npm install`
run `npm start`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166790041](https://www.pivotaltracker.com/story/show/166790041)
#### Screenshots (if appropriate)
Screenshot 2019-08-13 at 17 04 51

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+build/Release
+build
+.env
```
Comment (shaluchandwani) [inline .gitignore]: Please add dist folder in .gitignore

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+{     "presets": ["@babel/preset-env", "@babel/preset-react"] }
```
Comment (Joe-Joseph) [inline .babelrc]: Is this enough for us to be able to use es6 and async await?

File: .gitignore (hunk 1 above)
Comment (shaluchandwani) [reply .gitignore]: please add .DS_STORE also

**Review by kagabof — APPROVED** [review]
It looks fine.

**Review by shaluchandwani — APPROVED** [review]
2 changes were suggested and they were completed

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      }
+    ]
+  },
+  plugins: [
+    new HtmlWebPackPlugin({
+      template: "./src/index.html",
+      filename: "./index.html"
+    })
+  ]
+};
```
Comment (billkabanga) [inline webpack.config.js]: @mnzube Good work on the PR. Please also add a new line at the end of the file. Fix this for the other files as well.

Comment (mnzube) [reply webpack.config.js]: ok, doing that shortly

**Review by pngabo — APPROVED** [review]

File: .babelrc (hunk 2 above)
Comment (kagabof) [reply .babelrc]: Yes@babel/preset-env helps us use the latest JavaScript and that can allow us to use es6.

**Pull request merged**